src/plugins/discover/public/application/main/components/sidebar/__stories__/fields.ts

```diff
@@ -6,7 +6,8 @@
  * Side Public License, v 1.
  */
 
-import { FieldSpec, IndexPatternField } from '../../../../../../../data_views/common';
+import { FieldSpec } from '../../../../../../../data_views/common';
```

Comment:
This should be made available to `data_views/public` on `src/plugins/data_views/public/index.ts:25` - there may be other similar cases

Reply:
done